lib/board.rb

```diff
@@ -0,0 +1,29 @@
+require 'pry'
+class Board
+  attr_reader :board
+  def initialize
+    @board = board
+
+  end
+
+  def create_board
+  @board = Array.new(6) {Array.new(7) {"."}}
+    @board.unshift([*('A'..'G')])
```

Comment:
We will need an the array in the unshift to be a hash later and I was thinking of making it a hash now, but I think that would jeopardize how the board is displayed or some other method, so I am ok with leaving that for now. Just a though on how we can make our code more concise later.